eZ/Publish/Core/FieldType/Image/ImageThumbnailStrategy.php

```diff
@@ -43,12 +53,38 @@ public function getFieldTypeIdentifier(): string
 
     public function getThumbnail(Field $field, ?APIVersionInfo $versionInfo = null): ?Thumbnail
     {
-        /** @var \eZ\Publish\SPI\Variation\Values\ImageVariation $variation */
-        $variation = $this->variationHandler->getVariation(
-            $field,
-            $versionInfo ?? new VersionInfo(),
-            $this->variationName
-        );
+        try {
+            /** @var \eZ\Publish\SPI\Variation\Values\ImageVariation $variation */
+            $variation = $this->variationHandler->getVariation(
+                $field,
+                $versionInfo ?? new VersionInfo(),
+                $this->variationName
+            );
+        } catch (SourceImageNotFoundException $e) {
+            $this->logger->warning(
+                sprintf(
+                    'Thumbnail source image generated for %s field and %s variation could not be found (%s). %s',
+                    $field->fieldTypeIdentifier,
+                    $this->variationName,
+                    $e->getMessage(),
+                    $this->generateContentDetailsMessage($versionInfo)
+                )
+            );
```

Comment:
Pass exception to logger context, so this can be picked up by logging tools.
```suggestion
            $this->logger->warning(
                sprintf(
                    'Thumbnail source image generated for %s field and %s variation could not be found (%s). %s',
                    $field->fieldTypeIdentifier,
                    $this->variationName,
                    $e->getMessage(),
                    $this->generateContentDetailsMessage($versionInfo)
                ),
                [
                    'exception' => $e,
                ],
            );
```